### PR TITLE
feat: add display mode

### PR DESCRIPTION
**Pull request by thoa01**

### CODE REVIEW

**Review by DanHuy77 — APPROVED** [review]

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 1]
```diff
@@ -63,9 +64,16 @@ const JoinRoom = () => {
...
 
   const handleModeChange = (event) => {
-    setSelectedMode(event.target.value);
+    if (role === 1) {
+      setSelectedMode(event.target.value);
+    }
   };
 
+  useEffect(() => {
+    console.log(selectedMode);
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js]: Remove this

File: the-telephone-app-fe/src/components/ShowResult/ShowResult.js [hunk 2]
```diff
@@ -49,6 +47,7 @@ const ShowResult = () => {
         `user/result/${location.state?.data[player].nickname}/${location.state?.id_room}`
     );
     const responseResult = response.data;
+    console.log(response.data);
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/ShowResult/ShowResult.js]: Remove this

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 3]
```diff
@@ -22,6 +22,7 @@ import LoadingEffect from "../LoadingEffect/LoadingEffect";
 const JoinRoom = () => {
   const location = useLocation();
   const role = location.state?.role;
+  const modeCurrent = location.state?.modeCurrent;
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js]: should be currentMode

File: the-telephone-app-fe/src/components/StartGame/StartGame.js [hunk 4]
```diff
@@ -59,10 +59,12 @@ const StartGame = () => {
 
   var client = null;
   let mode;
+  let modeCurrent;
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/StartGame/StartGame.js]: should be currentMode

**Review by ces-manhnguyen — APPROVED** [review]

**Pull request merged**